### DISCUSSION TITLE
Download databases with MetGem 3.1.6

### DISCUSSION THREAD

**Discussion by SandraKTFC** [metgem/metgem#111] [Q&A] [answered]
Hello,
I don't succeed to download databases from external source; when I try I have the error message "Unhandled exception".
Do you know how to fix that (I am working with the version 3.1.6)?
Thank you,
Sandra

**Comment by n-elie** [comment, marked as answer]
Hello Sandra,

You have two options:

1. Use a more up-to-date but still unstable [development version](https://github.com/metgem/metgem/releases/tag/nightly) of MetGem
2. Update plugins using the `Plugins > Plugins Manager` menu:
![image](https://user-images.githubusercontent.com/40382614/227529490-bfcd1407-0ef7-4dce-b569-39b1a66d4e80.png)

Nicolas
